libplctagdevice.go

```diff
@@ -147,19 +146,19 @@ const (
 )
 
 func (dev *libplctagDevice) getID(tagName string) (C.int32_t, error) {
-	id, ok := dev.ids[tagName]
+	val, ok := dev.ids.Load(tagName)
 	if ok {
-		return id, nil
+		return val.(C.int32_t), nil
```

Comment:
Just so I understand, this is just explicitly casting the value out from the load?

Reply:
Yep. The result is either a `C.int32_t` or a panic.
Theres syntax to check if the type is correct, but I didn't bother because I know what I'm putting into that map.

Reply:
👍